BYT/View Controllers/FoaasViewController.swift

```diff
@@ -262,9 +262,13 @@ class FoaasViewController: UIViewController, FoaasViewDelegate, FoaasSettingMenu
     
     func camerarollButtonTapped() {
         print("cameraroll button tapped")
-        guard let vaidImage = getScreenShotImage(view: self.view) else { return }
+        guard let validImage = getScreenShotImage(view: self.view) else { return }
         //https://developer.apple.com/reference/uikit/1619125-uiimagewritetosavedphotosalbum
-        UIImageWriteToSavedPhotosAlbum(vaidImage, self, #selector(createScreenShotCompletion(image: didFinishSavingWithError: contextInfo:)), nil)
+        UIImageWriteToSavedPhotosAlbum(validImage, self, #selector(createScreenShotCompletion(image: didFinishSavingWithError: contextInfo:)), nil)
+        
+        //after the screenshot is saved, the settings menu will animate back into it's original position (show: true).
+        //this will give the false impression that the screenshot includes the settings menu because of how quickly it occurs.
+        animateSettingsMenu(show: true, duration: 0.1)
```

Comment:
I think I understand the logic behind this, but it feels hacky in that it seems the screenshot is being produced as a side effect of hoping that there is a delayed animation/drawing pass to create the screenshot. It looks like it'll work now, but I'm curious if that is still the case on different devices, *or* if we add more functionality to the screen grab that will cause the drawing/saving to take longer (like adding a watermark, which incurs additional overhead because of the alpha channels)